diff --git a/src/Main.cpp b/src/Main.cpp
index 5e35d4fd4622..c079ea343478 100644
--- a/src/Main.cpp
+++ b/src/Main.cpp
@@ -73,8 +73,6 @@ llvm::cl::opt<bool> FullExplore("full-explore",
         llvm::cl::desc("Terminate only when we tried everything in the search space."));
 llvm::cl::opt<bool> FullSynthesis("full-synthesis",
         llvm::cl::desc("Try all conditions instead of choosing the first one."));
-llvm::cl::opt<std::string> SummaryFile("summary-file", llvm::cl::init(""),
-        llvm::cl::desc("Output the list of generated patch id and scores to the file, sorted!"));
 llvm::cl::opt<bool> ConsiderAll("consider-all",
         llvm::cl::desc("Consider all possible files."));
 llvm::cl::opt<unsigned int> FirstNLoc("first-n-loc", llvm::cl::init(5000),
@@ -222,7 +220,6 @@ int main(int argc, char* argv[]) {
     E.setLocLimit(FirstNLoc.getValue());
     E.setGeoP(GeoP.getValue());
     E.setRandom(Random.getValue());
-    E.setSummaryFile(SummaryFile.getValue());
     if (Timeout.getValue() != 0)
         E.setTimeoutLimit(((unsigned long long)Timeout.getValue()) * 60 * 60);
 
